script.js

```diff
@@ -3,9 +3,16 @@ function sumValues()
     var N1 = parseFloat(document.getElementById('N1').value);
     var N2 = parseFloat(document.getElementById('N2').value);
     var oper = document.getElementById('operators').value;
-    var calculate;
+    
     if(oper === "+") {
-        calculate = N1 + N2;
+        var calculate = N1 + N2;}
+    else if(oper === "-") {
+        calculate = N1 - N2;}
```

Comment:
Great job! The answer key pushes the switch statement for this particular project but either are fine.  What did your bug end up being? That would be a fantastic thing to log in your programming notebook for if it ever happens in the future you will have the solution at hand instead of trying to remember.

Reply:
The fix for the bug was removing the <form> tags out where my button input was. It didn't work for a curious reason.

Reply:
the form tags